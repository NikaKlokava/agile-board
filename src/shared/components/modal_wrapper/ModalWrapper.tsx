import { ReactNode } from "react";
import cl from "./modal_wrapper.module.css";

type Props = {
  children: ReactNode;
};

export const ModalWrapper = ({ children }: Props) => {
  return (
    <div className={cl.modal_wrapper}>
      <div className={cl.modal_container}> {children}</div>
    </div>
  );
};
