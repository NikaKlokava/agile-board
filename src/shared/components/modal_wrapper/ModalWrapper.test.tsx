import renderer from "react-test-renderer";
import { ModalWrapper } from "./ModalWrapper";

describe("Test the ModalWrapper component", () => {
  test("The ModalWrapper renders correctly", () => {
    const modalWrapperSnap = renderer
      .create(<ModalWrapper children={undefined} />)
      .toJSON();
    expect(modalWrapperSnap).toMatchSnapshot();
  });
});
