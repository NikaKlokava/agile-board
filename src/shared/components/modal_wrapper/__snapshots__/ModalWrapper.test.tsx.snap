// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the ModalWrapper component The ModalWrapper renders correctly 1`] = `
Array [
  <div
    className="modal_wrapper"
  />,
  <div
    className="modal_container"
  >
     
  </div>,
]
`;
